tests/testthat/test-indent-mod.R

```diff
@@ -171,7 +171,7 @@ test_that("getters and setters work", {
   t0 <- basic_table() %>%
     summarize_row_groups("STUDYID", label_fstr = "overall summary") %>%
     split_rows_by("AEBODSYS", child_labels = "visible") %>%
-    summarize_row_groups("STUDYID", label = "subgroup summary") %>%
```

Comment:
Was this partial matching?

Reply:
yes - that's a perfect example of partial matching